api/projects.py

```diff
@@ -119,7 +119,7 @@ def get(self, uid=None, gid=None):
             if not group:
                 self.abort(400, 'invalid group id')
         query = {'group': gid} if gid else {}
-        projection = ['group', 'name', 'notes', 'timestamp', 'timezone']
+        projection = ['group', 'name', 'notes', 'timestamp', 'timezone', 'public']
```

Comment:
Is there a reason only the projection for projects is changing, and not Container or any other object?